### PR TITLE
Remove unnecessary workaround in favor of esperanto

### PR DESCRIPTION
**Pull request by tricknotes**

### CODE REVIEW

**Comment by rwjblue** [comment]
Bizarre failure seemingly unrelated to the changes. Restarting Travis...

File: packages/container/lib/container.js [hunk 1]
```diff
@@ -23,9 +20,6 @@ function Container(registry, options) {
   this._registry = registry || (function() {
     Ember.deprecate("A container should only be created for an already instantiated registry. For backward compatibility, an isolated registry will be instantiated just for this container.");
 
-    // TODO - See note above about transpiler import workaround.
-    if (!Registry) { Registry = requireModule('container/registry')['default']; }
```
Comment (rwjblue) [inline packages/container/lib/container.js]: Due to the requirement to be ES3 safe we do not get many of the features that allow cycles / late bindings.  Will have to revert this particular change.

Comment (rwjblue) [inline packages/container/lib/container.js]: #10340

Comment (tricknotes) [inline packages/container/lib/container.js]: Oops my bad :imp:

Comment (rwjblue) [inline packages/container/lib/container.js]: No worries!!! I wish it had worked, but I had a few teammates [nombom](https://github.com/rwjblue/dotfiles/blob/master/.zshrc#L52-L53) just after this was published and all hell broke loose in our tests....

Comment (tricknotes) [inline packages/container/lib/container.js]: Thanks dude. I won't be able to do without nombom... :innocent:

**Pull request merged**